app/models/user.rb

```diff
@@ -3,4 +3,6 @@ class User < ActiveRecord::Base
 
   validates :username, presence: true, length: { minimum: 6 }, uniqueness: true
   validates :email, presence: true, format: { with: /@/ }, uniqueness: true
+
+  enum role: ["default", "admin"]
```

Comment:
Same thing here. Don't really care which one you use but if you were to take hound into consideration you would do:

```ruby
  enum role: %w[default admin]
```